crates/common/tedge_config/src/tedge_config_cli/tedge_config_location.rs

```diff
@@ -102,6 +102,10 @@ impl TEdgeConfigLocation {
         Ok(TEdgeConfig::from_dto(&dto, self))
     }
 
+    pub fn load_dto_with_file_and_env(&self) -> Result<TEdgeConfigDto, TEdgeConfigError> {
```

Comment:
```suggestion
    pub fn load_dto_from_toml_and_env(&self) -> Result<TEdgeConfigDto, TEdgeConfigError> {
```

Reply:
fixed in https://github.com/thin-edge/thin-edge.io/pull/3326/commits/bfc6e14b0683c0cb3279a25d0148fd6c20cad744